search/src/KBMOSearch.cpp

```diff
@@ -142,59 +142,26 @@ std::vector<trajRegion> KBMOSearch::regionSearch(float xVel, float yVel, float r
     return res;
 }
 
-void KBMOSearch::clearPsiPhi() {
-    psiPhiGenerated = false;
-    psiImages = std::vector<RawImage>();
-    phiImages = std::vector<RawImage>();
-}
-
 void KBMOSearch::savePsiPhi(const std::string& path) {
     preparePsiPhi();
     saveImages(path);
 }
 
 void KBMOSearch::preparePsiPhi() {
     if (!psiPhiGenerated) {
-        startTimer("Preparing psi and phi images");
```

Comment:
Is the timing information no longer helpful for this method? Are you all done speeding it up? =]

Reply:
The way the code was written there is only one activate timer in the KBMOSearch object. Since we are using a timer outside this function, we want that to be the active one.